app/src/main/java/com/puzzlebench/clean_marvel_kotlin/presentation/CharacterFragment.kt

```diff
@@ -0,0 +1,62 @@
+package com.puzzlebench.clean_marvel_kotlin.presentation
+
+import android.app.Dialog
+import android.app.DialogFragment
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.puzzlebench.clean_marvel_kotlin.data.service.CharacterServicesImpl
+import com.puzzlebench.clean_marvel_kotlin.domain.usecase.GetCharacterByIdUseCase
+import com.puzzlebench.clean_marvel_kotlin.presentation.mvp.CharacterFragmentPresenter
+import com.puzzlebench.clean_marvel_kotlin.presentation.mvp.CharecterFragmentView
+
+// TODO: Rename parameter arguments, choose names that match
+// the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+private const val ARG_PARAM1 = "param1"
+private const val ARG_PARAM2 = "param2"
```

Comment:
private const should be in companion object

Reply:
Done! I remove the auto generated const, Thanks